static/openapi/prerendering-spec.json

```diff
@@ -1496,29 +809,1153 @@
+                              "description": "Date when the interaction was created."
+                            },
+                            "lastUpdated": {
+                              "type": "string",
+                              "format": "date-time",
+                              "description": "Date when the interaction was updated."
+                            },
+                            "config": {
+                              "type": "object",
+                              "properties": {
+                                "isActive": {
+                                  "type": "boolean",
+                                  "description": "Whether the interaction is active or not."
+                                },
+                                "global": {
+                                  "type": "array",
+                                  "items": {
+                                    "type": "object",
+                                    "properties": {
+                                      "isActive": {
+                                        "type": "boolean",
+                                        "description": "Whether the global interaction is active or not."
+                                      },
+                                      "htmlSelector": { "type": "string" },
+                                      "waitAfterInteraction": {
+                                        "type": "integer"
+                                      },
+                                      "executionSequence": { "type": "integer" }
+                                    },
+                                    "required": [
+                                      "isActive",
+                                      "htmlSelector",
+                                      "executionSequence"
+                                    ]
+                                  }
+                                },
+                                "specific": {
+                                  "type": "array",
+                                  "items": {
+                                    "type": "object",
+                                    "properties": {
+                                      "isActive": {
+                                        "type": "boolean",
+                                        "description": "Whether the specific interaction is active or not."
+                                      },
+                                      "htmlSelector": {
+                                        "type": "string",
+                                        "description": "HTML selector where the interaction is executed."
+                                      },
+                                      "waitAfterInteraction": {
+                                        "type": "integer",
+                                        "description": "Time to wait (in seconds) after the interaction execution."
```

Comment:
```suggestion
                                        "description": "Number of seconds to wait after the interaction executes."
```

Reply:
Changed to `Waiting time (in seconds) after the interaction is executed.`